data-science-onramp/ai-platform/modules/trainer/utils.py

```diff
@@ -34,8 +34,8 @@ def load_data(
 
     # [START ai_platform_utils_extract_ft]
     # Extract feature and target columns
-    feature_col = df.iloc[:, :-1]
-    target_col = df.iloc[:, -1:]
+    feature_col = df.iloc[:, :10]
+    target_col = df.iloc[:, 10:]
```

Comment:
Minor nit: I assume you'd want to only take the last column instead of 10 to the end?

This technically is the same if there are only 10 columns but could leave to inadvertent bugs if number of features change.

Why the change in any case?

Reply:
The target columns are sparse one-hot encoded vectors and only the first 10 columns are features. We provide these dimensions dynamically when creating the model.